router/index.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express');
-const { positionRouter } = require('./positions');
+const { workProfilesRouter } = require('./work-profiles.js');
 // const postulants = require('./postulants');
 const router = express.Router();
 
-router.use('/positions',positionRouter);
+router.use('/workProfiles',workProfilesRouter);
```

Comment:
Separate params with a space after the comma
```suggestion
router.use('/workProfiles', workProfilesRouter);
```

Reply:
Done. Sorry about that. 👎